File: sms-api-java/user-management-service/src/main/java/com/ffos/usermanagementservice/controller/dto/ApiResponse.java
package com.ffos.usermanagementservice.controller.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ApiResponse {
    private String code;
    private String message;
    private String docsURL;

    public ApiResponse() {

    }

    public ApiResponse(String code, String message, String docsURL) {
        this.code = code;
        this.message = message;
        this.docsURL = docsURL;
    }

}

File: sms-api-kotlin/userservice/src/main/kotlin/com/example/userservice/controller/dto/ApiResponse.kt
package com.example.userservice.controller.dto

import lombok.Getter
import lombok.Setter


@Getter
@Setter
class ApiResponse {
    var code: String? = null
    var message: String? = null
    var docsURL: String? = null

    constructor()
    constructor(code: String?, message: String?, docsURL: String?) {
        this.code = code
        this.message = message
        this.docsURL = docsURL
    }
}
